Monika After Story/game/zz_pianokeys.rpy

```diff
@@ -381,11 +381,21 @@ init -3 python in mas_piano_keys:
     import store
     import pygame # we need this for keymaps
     import os
-    log = store.mas_utils.getMASLog("log/pnm")
+    log = store.mas_logging.MASNewlineLogAdapter(store.mas_logging.init_log(
```

Comment:
Can we keep all the setup in `init_log`? Like how you can now pass in `formatter`, allow to pass in `adapter`? A bit weird that if you want a more custom setup you have to do something outside of init.